tests/test_other.py

```diff
@@ -713,6 +713,40 @@ def test_cmake_install(self):
     self.run_process(['cmake', '--build', 'build2'])
     self.assertContained('foo: 42\n', self.run_js('build2/Bar.js'))
 
+  def test_emcmake_cmake_commands(self):
```

Comment:
How about just calling this `test_cmake_commands` to match the test names above?   (I think its implied that `emake` tests are using `emmake` .. or at least the above tests seem to be named that way).

Reply:
I have renamed the test as desired (it seems to better fit the naming conventions, so thank you).